### PR TITLE
Docs on contributing to Corda

### PR DESCRIPTION
**Pull request by joeldudleyr3**

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Corda is an open-source project and we welcome contributions. This guide explains how to contribute back to Corda.
+
+.. contents::
+
+Identifying an area to contribute
+---------------------------------
+There are several ways to identify an area where you can contribute to Corda:
+
+* Browse open issues on the
```
Comment (gendal) [inline docs/source/contributing.rst]: We've talked for ages about a "help wanted" or "small niggly projects that we haven't got round to yet but which would be great for a newcomer as a way in" page... any progress?  How do we make it happen?  (Thanks for updating these docs btw)

Comment (joeldudleyr3) [reply docs/source/contributing.rst]: I knew you'd find this PR :)

It would be good, I'll speak to @GavT.

Comment (GavT) [reply docs/source/contributing.rst]: Can we export the contents of a JIRA issue filter to a dedicated corda.net page, i.e. Stories with a particular label such as "Help Wanted" are listed on this page and clicking on it takes you to the Story in Corda JIRA project. Does Wordpress have a JIRA export plugin?

Comment (GavT) [reply docs/source/contributing.rst]: Perhaps use this: https://wordpress.org/plugins/crane-app-jira-integration/#description

Comment (GavT) [reply docs/source/contributing.rst]: Or the standard landing page for Corda project on JIRA could be a dashboard showing Stories with "Help Wanted" listed

Comment (joeldudleyr3) [reply docs/source/contributing.rst]: Or we could just link to stories in a new "Help Wanted" epic? Seems more straightforward?

Comment (GavT) [reply docs/source/contributing.rst]: But the Epics server a different purpose in JIRA, i.e. delivery of key features. On reflection, we cannot have Epics being used for a different grouping purpose. So we'll need to use labels. We can link to a pre-built filter which lists Stories with "Help Wanted" label

**Comment by GavT** [comment]
I tested the filter in Chrome incognito mode and link does show CORDA stories labelled with HelpWanted. So this is good to go.

File: docs/source/contributing.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    contribution. Someone else may already be working on it, or it may be blocked by an on-going piece of work
+
+* Ask in the `Corda Slack channel <http://slack.corda.net/>`_
+
+Making the required changes
+---------------------------
+
+1. Create a fork of the master branch of the `Corda repo <https://github.com/corda/corda>`_
+2. Clone the fork to your local machine
+3. Make the changes, in accordance with the :doc:`code style guide </codestyle>`
```
Comment (GavT) [inline docs/source/contributing.rst]: This fails to render properly on actual page

**Pull request merged**